api/app/navigation.py

```diff
@@ -1,77 +1,56 @@
-import os
-from flask import Blueprint, request, jsonify
+from flask import Blueprint, jsonify, request
 from flask_cors import cross_origin
-from .graph.Graph2 import Graph
-from pathlib import Path
+from app.aiapi import AINavigationAPI
```

Comment:
This import is causing SonarQube tests to fail.
```suggestion
from api.app.aiapi import AINavigationAPI
```